src/Qir/Runtime/lib/Simulators/FullstateSimulator.cpp

```diff
@@ -230,30 +230,30 @@ namespace Quantum
             dump(this->simulatorId, callback);
         }
 
-        virtual std::string QubitToString(Qubit q) override
+        virtual std::string QubitToString(QubitIdType q) override
         {
             return std::to_string(GetQubitId(q));

```

Comment:
```suggestion
            return std::to_string(q);
```

Reply:
Good catch! While the `GetQubitId` is still needed for converting to `unsigned` when calling into the simulator, it doesn't need to be used here.